src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-from-with-a-publicly-accessible-standalone-devfile-using-http.adoc

```diff
@@ -0,0 +1,19 @@
+// configuring-a-workspace-using-a-devfile
+
+[id="creating-a-workspace-from-with-a-publicly-accessible-standalone-devfile-using-http_{context}"]
+= Creating a workspace from/with a publicly accessible standalone devfile using HTTP
```

Comment:
```suggestion
= Creating a workspace from a publicly accessible standalone devfile using HTTP
```
@MichalMaler rule of thumb: no slashes in docs text.

Reply:
Ah sorry, that is a Google doc planing residual. I wanted to ask what will be better for this heading but forget to delete it afterwards